pkg/envoy/registry/registry_test.go

```diff
@@ -40,5 +43,29 @@ var _ = Describe("Test catalog proxy register/unregister", func() {
 			connectedProxies := proxyRegistry.ListConnectedProxies()
 			Expect(len(connectedProxies)).To(Equal(0))
 		})
+
+		It("ensures the correctness of proxy count", func() {
```

Comment:
Not going to block on this, but would be nice to move away from ginko in unit tests

Reply:
+1 table driven tests or at least testify assert is more idiomatic

Reply:
Also a +1 on this! We are moving away from ginkgo for testing, so we shouldn't be introducing more at this point.